lib/model/store.dart

```diff
@@ -361,6 +369,8 @@ class PerAccountStore extends ChangeNotifier with StreamStore, MessageStore {
 
   final RecentDmConversationsView recentDmConversationsView;
 
+  final TypingStatus typingStatus;
```

Comment:
Hmm. I'm not sure this fits in the category "Messages, and summaries of messages". Maybe it's a better fit for "Users and data about them" and could go there?

Reply:
Yeah, neither is a perfect fit.

I like the "users" area a bit better, because that's definitely where presence data will naturally go, and this is a lot like presence.